src/pages/workspace/expensifyCard/WorkspaceCardsListLabel.tsx

```diff
@@ -0,0 +1,125 @@
+    const styles = useThemeStyles();
+    const {windowWidth} = useWindowDimensions();
+    const {shouldUseNarrowLayout} = useResponsiveLayout();
+    const theme = useTheme();
+    const {translate} = useLocalize();
+    const [bankAccountList] = useOnyx(ONYXKEYS.BANK_ACCOUNT_LIST);
+    const [isVisible, setVisible] = useState(false);
+    const [anchorPosition, setAnchorPosition] = useState({top: 0, left: 0});
+    const anchorRef = useRef(null);
+
+    const isConnectedWithPlaid = useMemo(() => !!Object.values(bankAccountList ?? {})[0]?.accountData?.additionalData?.plaidAccountID, [bankAccountList]);
+
+    useEffect(() => {
+        if (!anchorRef.current || !isVisible) {
+            return;
+        }
+
+        const position = getClickedTargetLocation(anchorRef.current);
+        const BOTTOM_MARGIN_OFFSET = 3;
+
+        setAnchorPosition({
+            top: position.top + position.height + BOTTOM_MARGIN_OFFSET,
+            left: position.left,
+        });
+    }, [isVisible, windowWidth]);
+
+    const requestLimitIncrease = () => {
+        // TODO: uncomment when RequestExpensifyCardLimitIncrease API call is supported
+        // Policy.requestExpensifyCardLimitIncrease(settlementBankAccountID);
+        setVisible(false);
+        Report.navigateToConciergeChat();
+    };
+
+    return (
+        <View style={styles.flex1}>
+            <View
+                ref={anchorRef}
+                style={[styles.flexRow, styles.alignItemsCenter, styles.mb1, style]}
+            >
+                <Text style={[styles.mutedNormalTextLabel, styles.mr1]}>{translate(`workspace.expensifyCard.${type}`)}</Text>
+                <PressableWithFeedback
+                    accessibilityLabel={translate(`workspace.expensifyCard.${type}`)}
+                    accessibilityRole={CONST.ROLE.BUTTON}
+                    onPress={() => setVisible(true)}
+                >
+                    <Icon
+                        src={Expensicons.Info}
+                        width={variables.fontSizeLabel}
+                        height={variables.fontSizeLabel}
+                        fill={theme.icon}
+                    />
+                </PressableWithFeedback>
+            </View>
+
+            <Text style={styles.shortTermsHeadline}>{CurrencyUtils.convertToDisplayString(value, 'USD')}</Text>
+
+            <Popover
+                onClose={() => setVisible(false)}
+                isVisible={isVisible}
+                outerStyle={!shouldUseNarrowLayout ? styles.pr5 : undefined}
+                innerContainerStyle={!shouldUseNarrowLayout ? {maxWidth: variables.modalContentMaxWidth} : undefined}
+                anchorRef={anchorRef}
+                anchorPosition={anchorPosition}
+            >
+                <View style={styles.p4}>
+                    <Text
+                        numberOfLines={1}
+                        style={[styles.optionDisplayName, styles.sidebarLinkTextBold, styles.mb2]}
```

Comment:
what does sidebarLinkTextBold do here?

Reply:
Oh, sorry, this style was used in several similar places, so it confused me I guess.
Could you provide me correct styles for this, so I double check it's okay

![image](https://github.com/Expensify/App/assets/23176449/131b5038-6e5e-4dea-adb6-b0295c8f77e8)

Reply:
Got it. I think all it does is make the text bold, and it adds a heading color which isn't needed since we just use our regular text color for that. 
![CleanShot 2024-06-27 at 19 12 59@2x](https://github.com/Expensify/App/assets/2319350/5e6c68ad-d165-46ad-9420-887e53674590)

So I wonder if you could simply just use `textStrong` and we'd be good?